pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -264,6 +272,7 @@ servingInfo:
   certFile: /var/serving-cert/tls.crt
   keyFile: /var/serving-cert/tls.key
 providers: {}
+proxy: {}
```

Comment:
Worth asking if we should start using `omitempty` on new fields so we don't have to have the empty structs.

Reply:
I guess we should... at least in our own structs